src/rules/no-ignored-unsubscribe.ts

```diff
@@ -0,0 +1,30 @@
+import type { TSESTree } from '@typescript-eslint/types';
+import { createRule } from "../utils";
+
+export default createRule("no-ignored-unsubscribe", {
+	meta: {
+		docs: {
+			description: "Forbids ignoring the unsubscribe method returned by the `subscribe()` os Svelte stores.",
+			category: 'Best Practices',
+			recommended: false,
+		},
+		fixable: undefined,
+		hasSuggestions: false,
+		messages: {
+			forbidden: "Ignoring returned value of the subscribe method is forbidden.",
+		},
+		schema: [],
+		type: "problem",
+	},
+	create: (context) => {
+		return {
+			"ExpressionStatement > CallExpression > MemberExpression[property.name='subscribe']":
```

Comment:
Add `foo(a.subscribe)` to test case. I think there is probably a false positive.

You can fix it by adding a property like this:

```suggestion
			"ExpressionStatement > CallExpression > MemberExpression.callee[property.name='subscribe']":
```


https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IFwibm8tcmVzdHJpY3RlZC1zeW50YXhcIjogW1wiZXJyb3JcIiwgXCJFeHByZXNzaW9uU3RhdGVtZW50ID4gQ2FsbEV4cHJlc3Npb24gPiBNZW1iZXJFeHByZXNzaW9uLmNhbGxlZVtwcm9wZXJ0eS5uYW1lPSdzdWJzY3JpYmUnXVwiXSAqL1xuXG5mb28oYS5zdWJzY3JpYmUpXG5hLnN1YnNjcmliZSgpIiwib3B0aW9ucyI6eyJlbnYiOnsiZXM2Ijp0cnVlfSwicnVsZXMiOnt9LCJwYXJzZXJPcHRpb25zIjp7ImVjbWFGZWF0dXJlcyI6e30sImVjbWFWZXJzaW9uIjoibGF0ZXN0Iiwic291cmNlVHlwZSI6InNjcmlwdCJ9fX0=